react/react1/week1/todolist/src/components/ItemRow.js

```diff
@@ -0,0 +1,16 @@
+import ItemDescription from "./ItemDescription";
+import Checkbox from "./Checkbox";
+import DeleteButton from "./DeleteButton";
+function ItemRow({ description, deadline, onDelete, changeStatus, className }) {
```

Comment:
ItemRow makes me think this is a row inside the item, you could use todo item instead